README.md

```diff
@@ -1,62 +1,58 @@
-:wave:	*New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:	
-
+:wave: *New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:
 
 ![Node.js CI](https://github.com/openmrs/openmrs-esm-patient-management/workflows/Node.js%20CI/badge.svg)
 
-
 # OpenMRS Patient Management
 
-This is a [Lerna](https://lerna.js.org/) project containing patient management frontend modules which deal with creating, searching, and listing patients. This package handles the following:
+This is a frontend module for the OpenMRS SPA. It bundles together various microfrontends that handle registering and editing patients, searching for existing patients, creating patient lists and managing patients in an outpatient setting. The microfrontends within this module include:
 
--  [@openmrs/esm-patient-search-app](packages/esm-patient-search-app)
--  [@openmrs/esm-patient-registration-app](packages/esm-patient-registration-app)
--  [@openmrs/esm-patient-list-app](packages/esm-patient-list-app)
--  [@openmrs/esm-outpatient-app](packages/esm-outpatient-app)
+-  [Patient search](packages/esm-patient-search-app)
+-  [Patient registration](packages/esm-patient-registration-app)
+-  [Patient list](packages/esm-patient-list-app)
+-  [Outpatient app](packages/esm-outpatient-app/README.md)
 
-## Repository Development
+## Setup
 
+Check out the developer documentation [here](http://o3-dev.docs.openmrs.org).
 
-### Prerequisites
+This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna).
 
-- [Node](https://nodejs.org/en/download)
-- yarn ```sh npm install yarn -g ```
-### Getting started
+To start a dev server for a specific microfrontend, run:
 
-To install and setup the repository once cloned just use the following command
-
-```sh
-npx lerna bootstrap
+```bash
+yarn start --sources 'packages/esm-<insert-package-name>-app'
 ```
 
-To develop on a specific package e.g [@openmrs/esm-patient-search-app](packages/esm-patient-search-app)
-
-```sh
-npx openmrs develop --sources 'packages/esm-patient-search-app'
-```
+This command uses the [openmrs](https://www.npmjs.com/package/openmrs) tooling to fire up a dev server running `esm-patient-chart` as well as the specified microfrontend.
 
-You can always use regex
+To start a dev server running all the packages, run:
 
-```sh
-npx openmrs develop --sources 'packages/esm-patient-{package1,package2}-app/'
+```bash
+yarn start-all
```

Comment:
This package doesn't have this command—only `start` which does what I would expect `start-all` to do.